tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -118,12 +120,12 @@ public function test_after_inline_script_with_delayed_main_script( $strategy ) {
 		$output    = get_echo( 'wp_print_scripts' );
 		$expected  = "<script type='text/javascript' src='http://example.org/ms-isa-1.js' id='ms-isa-1-js' data-wp-strategy='{$strategy}'></script>\n";
 		$expected .= wp_get_inline_script_tag(
-			"console.log(\"after one\");\n",
+			'console.log("after one");',
 			array(
 				'id' => 'ms-isa-1-js-after',
 			)
 		);
-		$this->assertSame( $expected, $output, 'Inline scripts in the "after" position, that are attached to a deferred main script, are failing to print/execute.' );
+		$this->assertEqualMarkup( $expected, $output, 'Inline scripts in the "after" position, that are attached to a deferred main script, are failing to print/execute.' );
```

Comment:
❤️ all of these. great.

also, did you know that we're finally going to get a [spec-compliant HTML5/DOM parser in PHP](https://wiki.php.net/rfc/domdocument_html5_parser)?

doesn't remove the need for the HTML API (mainly because of memory and performance and interface needs) but it will go a long way in our tests to eliminating false errors.